mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -136,6 +152,47 @@ AUI.add(
 							taskInput.val('');
 
 							instance._updateTaskCounts();
+
+							var resourceURL = taskInput.attr('data-resourceURL');
```

Comment:
You removed the data param, as requested in https://github.com/markocikos/liferay-internship/pull/169#discussion_r132224069, but you didn't implement passing through JS module config. Because of this, ```resourceURL``` is empty and the whole solution does nothing.

This is the main reason why I am not merging, the feature is not working. Other comments are more-or-less minor details.